hack/unwanted-dependencies.json

```diff
@@ -23,9 +23,17 @@
     }
   },
   "status": {
```

Comment:
this now shows not just which unwanted dependencies are still in use, but what they are referenced by, so we can see if that is growing or what upstreams we need to work with to eliminate calls

Reply:
Nice!